src/app.js

```diff
@@ -6,8 +6,350 @@ import _ from 'underscore';
+    //const reservation = new Reservation(reservationData, {url:postURL});
+    reservation.set({"url": `${postURL}` });
+    console.log(reservation.get('url'));
+    console.log(reservation);
+    if (reservation.isValid()) {
+      reservation.save({}, {
+        success: events.successfullReservationSave,
+        error: events.failedReservationSave,
+      });
+    } else {
+      console.log('NOT VALID RESERVATION DATA');
+      events.failedReservationSave(reservation, {errors: reservation.validate() });
+    }
+  },
+  successfullReservationSave(reservation, response) {
+    console.log('successfulReservationSave');
+    console.log(reservation);
+    console.log(response);
+    $('#status-messages ul').empty();
+    $('#status-messages ul').append(`<li>${reservation.get('name')} added!</li>`)
+    $('#status-messages').show();
+  },
+  failedReservationSave(reservation, response) {
+    console.log('failedReservationSave');
+    console.log(reservation);
+    console.log(response);
+    console.log(response.responseText);
+    //let JSONresponse = JSON.parse(response);
+    //console.log(JSONresponse);
+    //console.log(JSON.stringify(response));
+    //const responseJSON = JSON.stringify(response);
+    $('#status-messages ul').append(`<li>${reservation.get('name')} WAS NOT added!</li>`);
+    $('#status-messages ul').empty();
+    //responseJSON.forEach(function(data) {
+    //  $('#status-messages ul').append(`<li>${data}</li>`);
+    //});
+    $('#status-messages').show();
+  },
+
+};
+
+//POST RESERVATION INFO (RECEIVE INFO FROM FORM)
+const submitReservation = function submitReservation() {
+  $('#add-reservation-form').submit( function(e) {
+    e.preventDefault();
+    let form = document.createElement("form");
+    let tripID = $('#add-reservation-form').attr('trip-id');
+    console.log(`tripID from trip ul: ${tripID}`);
+    //$('#trip-details-container').attr('reservation-trip-id');
+    const url = 'https://ada-backtrek-api.herokuapp.com/trips/' + tripID + '/reservations/';
+    console.log(`URL: ${url}`);
+    const personName = $(this).serializeArray()[0].value;
+    const formData = $(this).serialize();
+    console.log(`formData: ${formData}`)
+
+    $.post(url, formData, (response) => {
+      console.log('Received POST response:');
+      console.log(response);
+      alert(`Reservation confirmed for ${personName}` );
+      console.log(`successfully posted reservation for ${personName}`)
+      console.log(response);
+    })
+    .fail(function(response){
+      console.log(response);
+      $('#status-messages ul').append('<li>Post was unsuccessful</li>')
+    })
+    .always(function(){
+      console.log('always even if we have success or failure');
+    });
+  });
+};
+
+//MODAL
+
+const modalAddTrip = function() {
+  const modal = $('#add-a-trip-form-container');
+  const openModal = $('#add-new-trip');
+  // const closeModal = $('.close');
+  const closeModal = document.getElementsByClassName("close")[0];
```

Comment:
With jQuery available, you generally should be using `document.` for selectors since the jQuery shorthand is much shorter.